pkg/apis/postgres-operator.crunchydata.com/v1beta1/pgbackrest_types.go

```diff
@@ -414,7 +414,7 @@ type RepoStatus struct {
 	ReplicaCreateBackupComplete bool `json:"replicaCreateBackupComplete,omitempty"`
 
 	// A hash of the required fields in the spec for defining an Azure, GCS or S3 repository,
-	// Utilizd to detect changes to these fields and then execute pgBackRest stanza-create
+	// Utilized to detect changes to these fields and then execute pgBackRest stanza-create
```

Comment:
why not "utilized"? Or a "." at the end of line 416?

Reply:
Yeah, this struck me as weird but I didn't have enough of an anything to be able to figure out what to do to it.

Between not being particularly familiar w/ the comment structure, or the code...

Reply:
Yeah, not your problem, I was just curious. Thx!